src/tests/intg/test_kcm.py

```diff
@@ -116,6 +116,11 @@ def kcm_teardown():
         except OSError as osex:
             if osex.errno == 2:
                 pass
+        try:
+            os.unlink(abs_sock_path)
+        except OSError as osex:
+            if osex.errno == 2:
```

Comment:
This code does nothing. We catch the exception and compare value - if it is 2, we pass, if not, we do nothing.

the same mistake is in the original code above.

Shouldn't it be:

      except OSError as osex:
            if osex.errno != 2:
                raise osex